src/views/admin/PersonManagementView.vue

```diff
@@ -110,31 +110,37 @@
 
   const statuses: Array<string> = ['Aktiv', 'Inaktiv'];
 
-  function getPaginatedPersonen(page: number): void {
+  async function getPaginatedPersonen(page: number): Promise<void> {
     searchFilterStore.personenPage = page;
-    personStore.getAllPersons({
+    await personStore.getAllPersons({
       offset: (searchFilterStore.personenPage - 1) * searchFilterStore.personenPerPage,
       limit: searchFilterStore.personenPerPage,
       organisationIDs: searchFilterStore.selectedSchulen || selectedSchulen.value,
       rolleIDs: searchFilterStore.selectedRollen || selectedRollen.value,
       searchFilter: searchFilterStore.searchFilter || searchFilter.value,
     });
+    await personenkontextStore.getAllPersonenuebersichten(
```

Comment:
Noch besser wäre, im Store zu kapseln, das zwei Aufrufe nötig sind.
`personenkontextStore.getAllPersonenuebersichten(filterObject)` und innerhalb des Stores werden die zwei Endpunkte getriggert. Das reduziert nebenbei die Code-Duplication in der Component.